apps/api/v1/routes/transactions.ts

```diff
@@ -106,7 +116,13 @@ const router = new Hono()
 
     const transaction = await createTransaction({
       user,
-      data,
+      data: {
+        ...data,
+        amount:
+          category?.type === 'INCOME'
+            ? Math.abs(data.amount)
+            : -Math.abs(data.amount),
```

Comment:
In case wallet account initials / adjusts, transaction is created with no category, and amount should be still positive.

Reply:
Left `amount` as is if no category is provided